drivers/wifi/nrf700x/src/wpa_supp_if.c

```diff
@@ -1779,6 +1779,8 @@ int nrf_wifi_supp_get_capa(void *if_priv, struct wpa_driver_capa *capa)
 		return -1;
 	}
 
+	memset(capa, 0, sizeof(*capa));
```

Comment:
I didn't get. Why should we sinitialize it. What if the caller has set some filed set then we aill over write it.

Reply:
Well, the owner of the `wpa_driver_capa` is Driver, so, it won't be filled by WPA supplicant.

Reply:
That I got about owner of capability.
I think memory is allocated by caller. it should be reponsible for it.'s initialization, 
we just need to use it.

Reply:
> I think memory is allocated by caller. it should be reponsible for it.'s initialization,

Caller ownes memory alloc/free, but contents are owned by this function, so, either it initialises or fills with some values etc is all part of the content population. And remember the initialisation is nothing new, it was there earlier as well, it was just moved.